utils/doclint/generateDotnetApi.js

```diff
@@ -94,7 +94,7 @@ let classNameMap;
 
   let writeFile = (name, out, folder) => {
     let content = template.replace('[CONTENT]', out.join(`${EOL}\t`));
-    fs.writeFileSync(`${path.join(folder, name)}.cs`, content);
```

Comment:
I assume they are renamed at some point later and it's not what end user will see, right?

Reply:
Yeah, this is not something that's ever visible to the end-user. It's just something that helps _us_ distinguish the generated files from everything else. :-)